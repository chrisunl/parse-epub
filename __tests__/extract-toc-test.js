import {
  contentOpf,
  entireHiddenTocHtml,
  hiddenItemTocHtml,
  hiddenSectionTocHtml,
  hiddenSingleItemTocHtml,
  nestedTocHtml,
  tocHtml
} from './data.js';
import extractManifest from '../extract/manifest.js';
import extractSpine from '../extract/spine.js';
import extractToc from '../extract/toc.js';

test('#extractToc', () => {
  const manifest = extractManifest(contentOpf)
  const spine = extractSpine(contentOpf)
  expect(extractToc(tocHtml, manifest, spine)).toMatchSnapshot();
});

test('#extractNestedToc', () => {
  const manifest = extractManifest(contentOpf)
  const spine = extractSpine(contentOpf)
  expect(extractToc(nestedTocHtml, manifest, spine)).toMatchSnapshot();
});

test('#extractEntireHiddenToc', () => {
  const manifest = extractManifest(contentOpf)
  const spine = extractSpine(contentOpf)
  expect(extractToc(entireHiddenTocHtml, manifest, spine)).toMatchSnapshot();
});

test('#extractHiddenSectionToc', () => {
  const manifest = extractManifest(contentOpf)
  const spine = extractSpine(contentOpf)
  expect(extractToc(hiddenSectionTocHtml, manifest, spine)).toMatchSnapshot();
});

test('#extractHiddenItemToc', () => {
  const manifest = extractManifest(contentOpf)
  const spine = extractSpine(contentOpf)
  expect(extractToc(hiddenItemTocHtml, manifest, spine)).toMatchSnapshot();
});

test('#extractHiddenSingleItemToc', () => {
  const manifest = extractManifest(contentOpf)
  const spine = extractSpine(contentOpf)
  expect(extractToc(hiddenSingleItemTocHtml, manifest, spine)).toMatchSnapshot();
});
