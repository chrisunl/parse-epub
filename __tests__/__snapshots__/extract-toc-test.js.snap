exports[`test #extractToc 1`] = `
Object {
  "byId": Object {
    "1": Object {
      "childNodes": Array [],
      "href": "s9ml/overview/table_of_contents.xhtml",
      "id": "1",
      "isLeaf": true,
      "label": undefined,
      "level": 1,
      "linear": undefined,
      "manifestId": "item-data-uuid-3b13d68a9f9447fe8bd6f2d7c7ea7403",
      "parentId": "__root__",
    },
    "10": Object {
      "childNodes": Array [],
      "href": "s9ml/culture_and_belonging/culture_and_belonging_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "10",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-6ad2f32297f84d1da04638f7e582c374",
      "parentId": "2",
    },
    "11": Object {
      "childNodes": Array [
        "12",
        "13",
        "14",
        "15",
        "16",
        "17",
        "18",
        "19",
      ],
      "href": "s9ml/the_thrill_of_horror/the_thrill_of_horror_opener.xhtml",
      "id": "11",
      "isLeaf": false,
      "label": undefined,
      "level": 1,
      "linear": undefined,
      "manifestId": "item-data-uuid-5adbebda98604da48e98180467c64cfd",
      "parentId": "__root__",
    },
    "12": Object {
      "childNodes": Array [],
      "href": "s9ml/the_thrill_of_horror/the_thrill_of_horror_opener.xhtml",
      "id": "12",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-5adbebda98604da48e98180467c64cfd",
      "parentId": "11",
    },
    "13": Object {
      "childNodes": Array [],
      "href": "s9ml/the_thrill_of_horror/its_a_mist_its_a_bat_its_a_._._._vampire.xhtml",
      "id": "13",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-9a306d7babff42c084fa63ad29079c97",
      "parentId": "11",
    },
    "14": Object {
      "childNodes": Array [],
      "href": "s9ml/the_thrill_of_horror/bringing_zombies_to_life.xhtml",
      "id": "14",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-7bf20da5723c4a5799c86f97c2fd81b3",
      "parentId": "11",
    },
    "15": Object {
      "childNodes": Array [],
      "href": "s9ml/the_thrill_of_horror/the_horror.xhtml",
      "id": "15",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-47a9d3e21464491cb37d718162b9b520",
      "parentId": "11",
    },
    "16": Object {
      "childNodes": Array [],
      "href": "s9ml/the_thrill_of_horror/the_open_window.xhtml",
      "id": "16",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-107c288bb253478f8a3094fdd57e00d0",
      "parentId": "11",
    },
    "17": Object {
      "childNodes": Array [],
      "href": "s9ml/the_thrill_of_horror/eeeek_how_to_scare_your_friends.xhtml",
      "id": "17",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-d15b805965d14521aeb48d93d168b65f",
      "parentId": "11",
    },
    "18": Object {
      "childNodes": Array [],
      "href": "s9ml/the_thrill_of_horror/scary_history_100_years_of_monster_movies.xhtml",
      "id": "18",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-0cf7f56918514bb784e1de2b5e739da1",
      "parentId": "11",
    },
    "19": Object {
      "childNodes": Array [],
      "href": "s9ml/the_thrill_of_horror/the_thrill_of_horror_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "19",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-3c0bb8315ea14a2bac12ace68240caff",
      "parentId": "11",
    },
    "2": Object {
      "childNodes": Array [
        "3",
        "4",
        "5",
        "6",
        "7",
        "8",
        "9",
        "10",
      ],
      "href": "s9ml/culture_and_belonging/culture_and_belonging_opener.xhtml",
      "id": "2",
      "isLeaf": false,
      "label": undefined,
      "level": 1,
      "linear": undefined,
      "manifestId": "item-data-uuid-d2a6e8f4db704bae875c9b4c5dc209cc",
      "parentId": "__root__",
    },
    "20": Object {
      "childNodes": Array [
        "21",
        "22",
        "23",
        "24",
        "25",
        "26",
        "27",
        "28",
      ],
      "href": "s9ml/the_move_toward_freedom/the_move_toward_freedom_opener.xhtml",
      "id": "20",
      "isLeaf": false,
      "label": undefined,
      "level": 1,
      "linear": undefined,
      "manifestId": "item-data-uuid-a4f73db0c4ff4464afdbdef6540a1feb",
      "parentId": "__root__",
    },
    "21": Object {
      "childNodes": Array [],
      "href": "s9ml/the_move_toward_freedom/the_move_toward_freedom_opener.xhtml",
      "id": "21",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-a4f73db0c4ff4464afdbdef6540a1feb",
      "parentId": "20",
    },
    "22": Object {
      "childNodes": Array [],
      "href": "s9ml/the_move_toward_freedom/a_final_escape.xhtml",
      "id": "22",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-ffc2e9c0ab5a404980a1c2686659ab6f",
      "parentId": "20",
    },
    "23": Object {
      "childNodes": Array [],
      "href": "s9ml/the_move_toward_freedom/frederick_douglass.xhtml",
      "id": "23",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-fb124defa1ef4c04a4f79397ae047df0",
      "parentId": "20",
    },
    "24": Object {
      "childNodes": Array [],
      "href": "s9ml/the_move_toward_freedom/from_the_red_badge_of_courage.xhtml",
      "id": "24",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-d43a2540b0774b5f973c841873a8b814",
      "parentId": "20",
    },
    "25": Object {
      "childNodes": Array [],
      "href": "s9ml/the_move_toward_freedom/from_reminiscences_of_my_life_in_camp.xhtml",
      "id": "25",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-14c6d82a58e64980992078f1726dac96",
      "parentId": "20",
    },
    "26": Object {
      "childNodes": Array [],
      "href": "s9ml/the_move_toward_freedom/freedom_spreads_through_the_states.xhtml",
      "id": "26",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-debfcf17707946f0bf220b7924adaa48",
      "parentId": "20",
    },
    "27": Object {
      "childNodes": Array [],
      "href": "s9ml/the_move_toward_freedom/the_freedom_train_keeps_rolling.xhtml",
      "id": "27",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-d1dc38f40de742b5a24353c05ed1a23e",
      "parentId": "20",
    },
    "28": Object {
      "childNodes": Array [],
      "href": "s9ml/the_move_toward_freedom/the_move_toward_freedom_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "28",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-ea0242f6d23d4e608c7252ad571882f5",
      "parentId": "20",
    },
    "29": Object {
      "childNodes": Array [
        "30",
        "31",
        "32",
        "33",
        "34",
        "35",
        "36",
        "37",
      ],
      "href": "s9ml/approaching_adulthood/approaching_adulthood_opener.xhtml",
      "id": "29",
      "isLeaf": false,
      "label": undefined,
      "level": 1,
      "linear": undefined,
      "manifestId": "item-data-uuid-17a19334765e441d848dd98a07d3b583",
      "parentId": "__root__",
    },
    "3": Object {
      "childNodes": Array [],
      "href": "s9ml/culture_and_belonging/culture_and_belonging_opener.xhtml",
      "id": "3",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-d2a6e8f4db704bae875c9b4c5dc209cc",
      "parentId": "2",
    },
    "30": Object {
      "childNodes": Array [],
      "href": "s9ml/approaching_adulthood/approaching_adulthood_opener.xhtml",
      "id": "30",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-17a19334765e441d848dd98a07d3b583",
      "parentId": "29",
    },
    "31": Object {
      "childNodes": Array [],
      "href": "s9ml/approaching_adulthood/the_birth_of_the_teenager.xhtml",
      "id": "31",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-0f371090c0e84cd09a06f868c3fe8efd",
      "parentId": "29",
    },
    "32": Object {
      "childNodes": Array [],
      "href": "s9ml/approaching_adulthood/jack_andraka_teen_scientist.xhtml",
      "id": "32",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-553be8a225384235b06f21698e403a16",
      "parentId": "29",
    },
    "33": Object {
      "childNodes": Array [],
      "href": "s9ml/approaching_adulthood/celeste_s_heart.xhtml",
      "id": "33",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-cc3d8d64c26046bc92151bfe70170c4c",
      "parentId": "29",
    },
    "34": Object {
      "childNodes": Array [],
      "href": "s9ml/approaching_adulthood/moving_forward_looking_back.xhtml",
      "id": "34",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-e8573f88150047078fdb5bfb083ca5c7",
      "parentId": "29",
    },
    "35": Object {
      "childNodes": Array [],
      "href": "s9ml/approaching_adulthood/old_enough_to_vote.xhtml",
      "id": "35",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-601228182dbc40e2ba71787be4c05654",
      "parentId": "29",
    },
    "36": Object {
      "childNodes": Array [],
      "href": "s9ml/approaching_adulthood/what_teenagers_need_to_thrive.xhtml",
      "id": "36",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-d11317bd1e344490a568604ca1f4616f",
      "parentId": "29",
    },
    "37": Object {
      "childNodes": Array [],
      "href": "s9ml/approaching_adulthood/approaching_adulthood_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "37",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-0f7b7bf0ed1b4c5c88b86bf81c8e2f92",
      "parentId": "29",
    },
    "38": Object {
      "childNodes": Array [
        "39",
        "40",
        "41",
        "42",
        "43",
        "44",
        "45",
        "46",
      ],
      "href": "s9ml/anne_franks_legacy/anne_frank_s_legacy_opener.xhtml",
      "id": "38",
      "isLeaf": false,
      "label": undefined,
      "level": 1,
      "linear": undefined,
      "manifestId": "item-data-uuid-ad4d37cb6ecb4c11ada0eafc3c44146f",
      "parentId": "__root__",
    },
    "39": Object {
      "childNodes": Array [],
      "href": "s9ml/anne_franks_legacy/anne_frank_s_legacy_opener.xhtml",
      "id": "39",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-ad4d37cb6ecb4c11ada0eafc3c44146f",
      "parentId": "38",
    },
    "4": Object {
      "childNodes": Array [],
      "href": "s9ml/culture_and_belonging/immigrants.xhtml",
      "id": "4",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-6d74e8e547ec455a9c00ebb37cb4a872",
      "parentId": "2",
    },
    "40": Object {
      "childNodes": Array [],
      "href": "s9ml/anne_franks_legacy/from_anne_frank_remembered.xhtml",
      "id": "40",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-3a5daf183fc34c31822a2b16d13a57ba",
      "parentId": "38",
    },
    "41": Object {
      "childNodes": Array [],
      "href": "s9ml/anne_franks_legacy/the_hitler_youth.xhtml",
      "id": "41",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-61380725c0dd484593df2807e84cf853",
      "parentId": "38",
    },
    "42": Object {
      "childNodes": Array [],
      "href": "s9ml/anne_franks_legacy/a_star_who_s_anne_s_age.xhtml",
      "id": "42",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-445da3f24e8d49d897c41c17a615bcef",
      "parentId": "38",
    },
    "43": Object {
      "childNodes": Array [],
      "href": "s9ml/anne_franks_legacy/dear_anne.xhtml",
      "id": "43",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-74687195b0fb413bba6b9ca5d8aa3cec",
      "parentId": "38",
    },
    "44": Object {
      "childNodes": Array [],
      "href": "s9ml/anne_franks_legacy/seven_reasons_to_keep_a_diary.xhtml",
      "id": "44",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-757f293b2be647deaa1c5291da00413e",
      "parentId": "38",
    },
    "45": Object {
      "childNodes": Array [],
      "href": "s9ml/anne_franks_legacy/the_legacy_lives_on.xhtml",
      "id": "45",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-f7923d60c5894cc999346a9f2c8f5ae7",
      "parentId": "38",
    },
    "46": Object {
      "childNodes": Array [],
      "href": "s9ml/anne_franks_legacy/anne_frank_s_legacy_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "46",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-101710cd602a4fcfa90108c5c0347df7",
      "parentId": "38",
    },
    "47": Object {
      "childNodes": Array [
        "48",
        "49",
        "50",
        "51",
        "52",
        "53",
        "54",
        "55",
      ],
      "href": "s9ml/the_value_of_work/the_value_of_work_opener.xhtml",
      "id": "47",
      "isLeaf": false,
      "label": undefined,
      "level": 1,
      "linear": undefined,
      "manifestId": "item-data-uuid-8fc059832a1f4c64be19490536ebc955",
      "parentId": "__root__",
    },
    "48": Object {
      "childNodes": Array [],
      "href": "s9ml/the_value_of_work/the_value_of_work_opener.xhtml",
      "id": "48",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-8fc059832a1f4c64be19490536ebc955",
      "parentId": "47",
    },
    "49": Object {
      "childNodes": Array [],
      "href": "s9ml/the_value_of_work/i_hear_america_singing.xhtml",
      "id": "49",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-cc1ee917649f4f55855784812d45ef9d",
      "parentId": "47",
    },
    "5": Object {
      "childNodes": Array [],
      "href": "s9ml/culture_and_belonging/selena_s_blog.xhtml",
      "id": "5",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-5b23866eff924613af22d8a296c61653",
      "parentId": "2",
    },
    "50": Object {
      "childNodes": Array [],
      "href": "s9ml/the_value_of_work/dawn_to_dusk_a_child_s_workday.xhtml",
      "id": "50",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-3a07ddda3070483e99e2b20349853c76",
      "parentId": "47",
    },
    "51": Object {
      "childNodes": Array [],
      "href": "s9ml/the_value_of_work/the_conduit_mouse.xhtml",
      "id": "51",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-bda8bcd0fdaa4c26be8914772d6c84ec",
      "parentId": "47",
    },
    "52": Object {
      "childNodes": Array [],
      "href": "s9ml/the_value_of_work/harnessing_the_wind.xhtml",
      "id": "52",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-589d9984380e4568b1eb5710e6e81ca2",
      "parentId": "47",
    },
    "53": Object {
      "childNodes": Array [],
      "href": "s9ml/the_value_of_work/weird_jobs_around_the_world.xhtml",
      "id": "53",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-1695c0869dbf4fc38e255f1d8571b4e5",
      "parentId": "47",
    },
    "54": Object {
      "childNodes": Array [],
      "href": "s9ml/the_value_of_work/take_a_break_and_other_ways_to_get_more_work_done.xhtml",
      "id": "54",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-7841571fa25b470fa9b4158ce0544f87",
      "parentId": "47",
    },
    "55": Object {
      "childNodes": Array [],
      "href": "s9ml/the_value_of_work/the_value_of_work_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "55",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-ec172a4c778d409da1aec2f5474cc337",
      "parentId": "47",
    },
    "6": Object {
      "childNodes": Array [],
      "href": "s9ml/culture_and_belonging/family_reunion.xhtml",
      "id": "6",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-40c45e21a58c438283bf95dcaa809537",
      "parentId": "2",
    },
    "7": Object {
      "childNodes": Array [],
      "href": "s9ml/culture_and_belonging/immigration_and_migration_in_the_united_states.xhtml",
      "id": "7",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-999b44bf3f03460cbdf41781bd702168",
      "parentId": "2",
    },
    "8": Object {
      "childNodes": Array [],
      "href": "s9ml/culture_and_belonging/what_is_a_culture.xhtml",
      "id": "8",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-01eee42463c54f1b945502ebf51ab9ab",
      "parentId": "2",
    },
    "9": Object {
      "childNodes": Array [],
      "href": "s9ml/culture_and_belonging/welcome_to_california.xhtml",
      "id": "9",
      "isLeaf": true,
      "label": undefined,
      "level": 2,
      "linear": undefined,
      "manifestId": "item-data-uuid-d17e58059b0542f1a831de1593127652",
      "parentId": "2",
    },
    "__root__": Object {
      "childNodes": Array [
        "1",
        "2",
        "11",
        "20",
        "29",
        "38",
        "47",
      ],
      "href": undefined,
      "id": "__root__",
      "isLeaf": false,
      "label": undefined,
      "level": 0,
      "linear": undefined,
      "manifestId": undefined,
      "parentId": undefined,
    },
  },
  "byManifestId": Object {
    "item-data-uuid-01eee42463c54f1b945502ebf51ab9ab": "8",
    "item-data-uuid-0cf7f56918514bb784e1de2b5e739da1": "18",
    "item-data-uuid-0f371090c0e84cd09a06f868c3fe8efd": "31",
    "item-data-uuid-0f7b7bf0ed1b4c5c88b86bf81c8e2f92": "37",
    "item-data-uuid-101710cd602a4fcfa90108c5c0347df7": "46",
    "item-data-uuid-107c288bb253478f8a3094fdd57e00d0": "16",
    "item-data-uuid-14c6d82a58e64980992078f1726dac96": "25",
    "item-data-uuid-1695c0869dbf4fc38e255f1d8571b4e5": "53",
    "item-data-uuid-17a19334765e441d848dd98a07d3b583": "30",
    "item-data-uuid-3a07ddda3070483e99e2b20349853c76": "50",
    "item-data-uuid-3a5daf183fc34c31822a2b16d13a57ba": "40",
    "item-data-uuid-3b13d68a9f9447fe8bd6f2d7c7ea7403": "1",
    "item-data-uuid-3c0bb8315ea14a2bac12ace68240caff": "19",
    "item-data-uuid-40c45e21a58c438283bf95dcaa809537": "6",
    "item-data-uuid-445da3f24e8d49d897c41c17a615bcef": "42",
    "item-data-uuid-47a9d3e21464491cb37d718162b9b520": "15",
    "item-data-uuid-553be8a225384235b06f21698e403a16": "32",
    "item-data-uuid-589d9984380e4568b1eb5710e6e81ca2": "52",
    "item-data-uuid-5adbebda98604da48e98180467c64cfd": "12",
    "item-data-uuid-5b23866eff924613af22d8a296c61653": "5",
    "item-data-uuid-601228182dbc40e2ba71787be4c05654": "35",
    "item-data-uuid-61380725c0dd484593df2807e84cf853": "41",
    "item-data-uuid-6ad2f32297f84d1da04638f7e582c374": "10",
    "item-data-uuid-6d74e8e547ec455a9c00ebb37cb4a872": "4",
    "item-data-uuid-74687195b0fb413bba6b9ca5d8aa3cec": "43",
    "item-data-uuid-757f293b2be647deaa1c5291da00413e": "44",
    "item-data-uuid-7841571fa25b470fa9b4158ce0544f87": "54",
    "item-data-uuid-7bf20da5723c4a5799c86f97c2fd81b3": "14",
    "item-data-uuid-8fc059832a1f4c64be19490536ebc955": "48",
    "item-data-uuid-999b44bf3f03460cbdf41781bd702168": "7",
    "item-data-uuid-9a306d7babff42c084fa63ad29079c97": "13",
    "item-data-uuid-a4f73db0c4ff4464afdbdef6540a1feb": "21",
    "item-data-uuid-ad4d37cb6ecb4c11ada0eafc3c44146f": "39",
    "item-data-uuid-bda8bcd0fdaa4c26be8914772d6c84ec": "51",
    "item-data-uuid-cc1ee917649f4f55855784812d45ef9d": "49",
    "item-data-uuid-cc3d8d64c26046bc92151bfe70170c4c": "33",
    "item-data-uuid-d11317bd1e344490a568604ca1f4616f": "36",
    "item-data-uuid-d15b805965d14521aeb48d93d168b65f": "17",
    "item-data-uuid-d17e58059b0542f1a831de1593127652": "9",
    "item-data-uuid-d1dc38f40de742b5a24353c05ed1a23e": "27",
    "item-data-uuid-d2a6e8f4db704bae875c9b4c5dc209cc": "3",
    "item-data-uuid-d43a2540b0774b5f973c841873a8b814": "24",
    "item-data-uuid-debfcf17707946f0bf220b7924adaa48": "26",
    "item-data-uuid-e8573f88150047078fdb5bfb083ca5c7": "34",
    "item-data-uuid-ea0242f6d23d4e608c7252ad571882f5": "28",
    "item-data-uuid-ec172a4c778d409da1aec2f5474cc337": "55",
    "item-data-uuid-f7923d60c5894cc999346a9f2c8f5ae7": "45",
    "item-data-uuid-fb124defa1ef4c04a4f79397ae047df0": "23",
    "item-data-uuid-ffc2e9c0ab5a404980a1c2686659ab6f": "22",
  },
  "items": Array [
    "1",
    "3",
    "4",
    "5",
    "6",
    "7",
    "8",
    "9",
    "10",
    "12",
    "13",
    "14",
    "15",
    "16",
    "17",
    "18",
    "19",
    "21",
    "22",
    "23",
    "24",
    "25",
    "26",
    "27",
    "28",
    "30",
    "31",
    "32",
    "33",
    "34",
    "35",
    "36",
    "37",
    "39",
    "40",
    "41",
    "42",
    "43",
    "44",
    "45",
    "46",
    "48",
    "49",
    "50",
    "51",
    "52",
    "53",
    "54",
    "55",
  ],
}
`;
